[2022-10-09T23:14:40.316+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cemaco.colector scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-10-09T23:14:40.324+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cemaco.colector scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-10-09T23:14:40.324+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-09T23:14:40.324+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-10-09T23:14:40.324+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-09T23:14:40.336+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): colector> on 2022-08-11 00:00:00+00:00
[2022-10-09T23:14:40.339+0000] {standard_task_runner.py:54} INFO - Started process 347 to run task
[2022-10-09T23:14:40.341+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cemaco', 'colector', 'scheduled__2022-08-11T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpiytveuia']
[2022-10-09T23:14:40.342+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask colector
[2022-10-09T23:14:40.342+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dag.py
[2022-10-09T23:14:40.593+0000] {task_command.py:384} INFO - Running <TaskInstance: cemaco.colector scheduled__2022-08-11T00:00:00+00:00 [running]> on host 7dcbfa2899a4
[2022-10-09T23:14:40.648+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cemaco
AIRFLOW_CTX_TASK_ID=colector
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-11T00:00:00+00:00
[2022-10-09T23:14:40.651+0000] {tasks.py:234} INFO - Reading prep dataframe
[2022-10-09T23:14:40.656+0000] {tasks.py:115} INFO - Instantiating the Browser with the requested settings
[2022-10-09T23:14:40.666+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks.py", line 236, in colector
    browser = config_browser(config)
  File "/opt/airflow/dags/tasks.py", line 118, in config_browser
    desired_capabilities=config['browser_arg'])
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 270, in __init__
    self.start_session(capabilities, browser_profile)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 428, in execute
    self.error_handler.check_response(response)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 243, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidArgumentException: Message: json: cannot unmarshal number into Go struct field Caps.capabilities.alwaysMatch.browserVersion of type string

[2022-10-09T23:14:40.676+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=cemaco, task_id=colector, execution_date=20220811T000000, start_date=20221009T231440, end_date=20221009T231440
[2022-10-09T23:14:40.685+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 12 for task colector (Message: json: cannot unmarshal number into Go struct field Caps.capabilities.alwaysMatch.browserVersion of type string
; 347)
[2022-10-09T23:14:40.714+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-10-09T23:14:40.728+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
